lib/index-finance.js

```diff
@@ -18,7 +18,8 @@ Plotly.register([
     require('./ohlc'),
     require('./candlestick'),
     require('./funnel'),
-    require('./waterfall')
+    require('./waterfall'),
+    require('./indicator')
```

Comment:
What is the best drawing order for this trace?
Should it be on top of everything else?

Reply:
I'd say `indicator` traces should be above all other traces.

Right now,

https://github.com/plotly/plotly.js/pull/3978/files#diff-2941ab69a12080c0633ff4ac8ea3aa83R3783

@antoinerg has it above all other SVG traces but below the WebGL traces. Perhaps we should place `_indicatorlayer` in `fullLayout._toppaper` next to colorbars, legends, updatemenus and sliders? This would be the first ever trace in `_toppaper` :smirk:

Reply:
Done in 03cb7c3b9b4fbd8cb29889457e0399363bb0c1cf with and :lock: with image test

Reply:
Nice. I'm glad that went smoothly!